examples_tests/13.BRDF_Explorer/BRDFExplorerApp.cpp

```diff
@@ -506,29 +506,35 @@ void BRDFExplorerApp::eventAOTextureBrowse(const ::CEGUI::EventArgs&)
         const auto image = ext::cegui::loadImage(p.second.c_str());
         loadTextureSlot(ETEXTURE_SLOT::TEXTURE_AO, image.buffer, image.w, image.h);
 
+        char buffer[1024];
+        sprintf(buffer, "%s (%ix%i)\nLeft-click to select a new texture.",
```

Comment:
sprintf is really unportable between compilers, use ostringstream

Reply:
There, better?